shared/vendor/js/jquery-plugins/jquery.autoSuggest.js

```diff
@@ -92,6 +93,7 @@
         var request = null;
         if(typeof data == "string" || typeof data == "function") {
             d_fetcher = function(query, next) {
+                query = $.trim(query);
```

Comment:
I have no idea what `data` does underneath and how it feeds into the baseUrl, but the plugin appears to be using query for the query parameter.